### PR TITLE
ref(invite): Backfill OrganizationMember invite_status

### PR DESCRIPTION
**Pull request by maheskett**
Backfill null OrganizationMember invite_status to `InviteStatus.APPROVED`

Follow-up to #14553

### CODE REVIEW

File: src/sentry/south_migrations/0506_backfill_organizationmember_invite_status.py [hunk 1]
```diff
@@ -0,0 +1,1482 @@
...
+    def _forwards(self, orm):
+        "Write your forwards methods here."
+        # Note: Don't use "from appname.models import ModelName".
+        # Use orm.ModelName to refer to models in this application,
+        # and orm['appname.ModelName'] for models in other applications.
+
+        for member in RangeQuerySetWrapperWithProgressBar(
+            orm.OrganizationMember.objects.filter(invite_status__isnull=True)
+        ):
+            member.invite_status = 0
```
Comment (mattrobenolt) [inline src/sentry/south_migrations/0506_backfill_organizationmember_invite_status.py]: I'd suggest doing this as:

```python
orm.OrganizationMember.objects.filter(id=member.id, invite_status__isnull=True).update(invite_status=0)
```

**Comment by mattrobenolt** [comment]
I'm alsog oing to recommend we do this offline, so flipping the `is_dangerous=True` flag. There are a lot of rows here, and this migration would take a long time to run.

**Review by mattrobenolt — DISMISSED** [review]
.

**Comment by wedamija** [comment]
There are only 6 rows to update here, it should be safe to do it online.

I was pretty surprised to see that when we added the column we actually set it to have a default of 0, and it has already been applied to most rows in the table.  And unless I'm reading the logs wrong, in the deploy where it ran it took less than a second to add that default to ~420k rows: https://freight.getsentry.net/deploys/getsentry/production/7330

**Comment by mattrobenolt** [comment]
We added a database level default to the column? If so, that locked the entire table and updated them all at once. Which is why it was fast. It wasn't doing 400,000 queries sequentially.

But I guess given this is only actually updating 6 rows here, then we're fine. I was expecting this to update them all.

Would still like to see the chance as `.filter().update()` instead for safety.

**Comment by wedamija** [comment]
> We added a database level default to the column? If so, that locked the entire table and updated them all at once. Which is why it was fast. It wasn't doing 400,000 queries sequentially.
> 
> But I guess given this is only actually updating 6 rows here, then we're fine. I was expecting this to update them all.
> 
> Would still like to see the chance as `.filter().update()` instead for safety.

Interesting, I guess it just rewrote the whole table, which is why it was so fast? That's cool, not so great that we locked the table though. At least it didn't seem to cause any downtime this time around. Agree on `filter().update`

**Review by wedamija — APPROVED** [review]

**Comment by maheskett** [comment]
Dang didn’t mean to set the default in the migration. Won’t do that again. Anyway, thanks for all the info.

**Pull request merged**